pipenv/core.py

```diff
@@ -1656,33 +1710,17 @@ def pip_install_deps(
         ]
         pip_args = get_pip_args(
             project,
-            pre=pre,
+            pre=project.settings.get("allow_prereleases", False),
```

Comment:
Same as above: If this is this something which is part of the Pipefile, then it should be documented in plette.

Reply:
I made a ticket in plette to handle any documentation updates there separately -- I have some idea that we would document the categories like `pipenv` but not the specific pipenv settings that can be supplied to the category, which are already documented in pipenv.   Let's not let it block this fix though:  https://github.com/sarugaku/plette/issues/34